src/assets/dto/redeem-tokens.dto.ts

```diff
@@ -16,4 +16,14 @@ export class RedeemTokensDto extends TransactionBaseDto {
   @ToBigNumber()
   @IsBigNumber()
   readonly amount: BigNumber;
+
+  @ApiProperty({
+    description:
+      'Portfolio number from which the Asset tokens must be redeemed. Use 0 for the Default Portfolio',
```

Comment:
```suggestion
      'Portfolio id from which the Asset tokens must be redeemed from. Use 0 for the Default Portfolio',
```
I think id is more consistent than number.

Reply:
We are using `number` at all places.